docs/components/switch/error-switch-demo.js

```diff
@@ -0,0 +1,28 @@
+import Util from '../../_util';
+
+if (document.getElementById('vue-errorSwitchDemo')) {
+    new Vue({
+        el: '#vue-errorSwitchDemo',
+        data: {
+            isInValid: false,
+            labelState: '',
+            option : 'ON',
+        },
+        computed: {
+            attrInValid: function () {
+                return (this.isInValid ? 'inValid' : '');
+            },
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-switch-control class="switch">
+                        <input type="checkbox" id="errorChkDemo" ${this.isInValid ? 'inValid' : '' }/>
```

Comment:
The `invalid` attribute should be all lowercase.

```js
 <input type="checkbox" id="errorChkDemo" ${this.isInValid ? 'invalid' : '' }/>
```

Reply:
Changes have been done, please review.